tasks/kernel.yml

```diff
@@ -26,29 +26,34 @@
   when: selinux_config.stat.exists
 
 # Limits Tunning
-- name: Add ulimit into /etc/init.d/functions 
-  lineinfile: 
+- name: Add ulimit into /etc/init.d/functions
+  lineinfile:
     dest=/etc/init.d/functions
     insertafter='^export PATH'
     line='ulimit -n 65536'
     state=present
   when: ansible_os_family == 'RedHat'
 
-## Haven't been tested yet.
-#- name: Add ulimit into /lib/lsb/init-functions
-#  lineinfile:
-#    dest=/lib/lsb/init-functions
-#    insertafter='^export PATH'
-#    line='ulimit -n 65536'
-#    state=present
-#  when: ansible_os_family == 'Debian'
+# see how to set default limit system-wide for systemd units
+# https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html
+# https://serverfault.com/questions/958760/changing-ulimit-on-systemd
+- name: 'default open file limit for systemd units'
+  template:
+    dest: /etc/systemd/system.conf.d/openfile.conf
```

Comment:
Another suggestion is to prefix all filenames in those subdirectories with a two-digit number and a dash, to simplify the ordering of the files.

https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#:~:text=It%20is%20recommended%20to%20prefix%20all%20filenames%20in%20those%20subdirectories%20with%20a%20two%2Ddigit%20number%20and%20a%20dash%2C%20to%20simplify%20the%20ordering%20of%20the%20files.

But I think it's okay as it currently stands. Leave the decision to you.

Reply:
good catch, updated.